File: EventCheckin.Api/Models/AssignPassModel.cs
using EventCheckin.Api.Models.Events;
using System.Collections.Generic;

namespace EventCheckin.Api.Models
{
    public class AssignPassModel
    {
        public AssignPassModel() 
        {
            Events = new List<EventModel>();
            TicketPasses = new List<TicketPassModel>();
        }

        public long EventId { get; set; }
        public List<TicketPassModel> TicketPasses { get; set;}

        public List<EventModel> Events { get; set; }
    }
}

File: EventCheckin.ClientApp/src/app/api/models/assign-pass-model.ts
/* tslint:disable */
/* eslint-disable */
import { EventModel } from '../models/event-model';
import { TicketPassModel } from '../models/ticket-pass-model';
export interface AssignPassModel {
  eventId?: number;
  events?: Array<EventModel> | null;
  ticketPasses?: Array<TicketPassModel> | null;
}
